packages/web/src/server.tsx

```diff
@@ -66,7 +135,9 @@ async function pipeS3Object(objectKey: string, res: Response) {
       throw new Error('Unexpected response format');
     }
   } catch (e) {
-    if (e instanceof NoSuchKey) {
+    if (fallback) {
+      res.send(fallback);
```

Comment:
As we discussed, placing the fallback client-side would save us and users minor amount of traffic. And, more so, be a step towards correct implementation.